includes/Order/Manager.php

```diff
@@ -30,9 +30,12 @@ public function all( $args = [] ) {
 
         $args = wp_parse_args( $args, $default );
 
-        $offset       = ( $args['paged'] - 1 ) * $args['limit'];
+        $offset      = ( $args['paged'] - 1 ) * $args['limit'];
         $cache_group = 'dokan_seller_data_'.$args['seller_id'];
-        $cache_key   = 'dokan-seller-orders-' . $args['status'] . '-' . $args['seller_id'];
+        
+        // Use all arguments to create a hash used as cache key
+        $cache_key   = 'dokan_seller_orders-' . md5(json_encode($args));
```

Comment:
@kvasbo Thanks for the PR. Seems good to me, just a minor change is needed. Can you please insert spaces as needed? Something like this.
```php
        $cache_key = 'dokan_seller_orders-' . md5( json_encode( $args ) );
```

Reply:
Too late now I guess :)